tests/integration/test_remoted/test_rids/test_config.py

```diff
@@ -0,0 +1,78 @@
+"""
+ Copyright (C) 2015-2021, Wazuh Inc.
+ Created by Wazuh, Inc. <info@wazuh.com>.
+ This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+"""
+
+import pytest
+
+from pathlib import Path
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils.services import control_service
+
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+from wazuh_testing.modules.remoted.configuration import REMOTED_DEBUG, REMOTED_WORKER_POOL, REMOTED_VERIFY_MSG_ID
```

Comment:
These `wazuh_testing` imports must be moved to the top with the rest of the same kind imports (move imports from lines: 17, 18, 19 to line 14)